frontend/constants/serviceTemplates.ts

```diff
@@ -79,18 +79,6 @@ export const PREDICT_SERVICE_TEMPLATE: ServiceTemplate = {
       value: '',
       provision_type: EnvProvisionType.COMPUTED,
     },
-    REQUESTER_STAKING_INSTANCE_ADDRESS: {
-      name: 'Requester staking instance address',
-      description: '',
-      value: '',
-      provision_type: EnvProvisionType.COMPUTED,
-    },
-    PRIORITY_MECH_ADDRESS: {
-      name: 'Priority Mech address',
-      description: '',
-      value: '',
-      provision_type: EnvProvisionType.COMPUTED,
-    },
```

Comment:
@Tanya-atatakai what will happen with these? They are no longer used by the agent.

Reply:
it'll remain in existing services, also I guess we can't use patch to remove variables? we set only those we want to add/update